### PR TITLE
CS-21: Support ruby 3.2 in CI and Docker

### PR DESCRIPTION
**Pull request by nazar-kostyk**
## What was done as a part of this PR?

- Add ruby 3.2 support to CI and Docker

## How it was done?

- Updated `ci.yml`
- Updated `Taskfile.yml`
- Added `docker/3.2/Dockerfile`
- Pointed `appraisal` gem to the thoughtbot fork to fix the `Errno::ENOENT` issue

## Checklist:

- [x] I have performed a self-review of my code.
- [x] I have added/adjusted tests that prove my fix is effective or that my feature works.
- [x] CI is green.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -4,3 +4,7 @@ source "https://rubygems.org"
 
 # Specify your gem's dependencies in convenient_service.gemspec
 gemspec
+
+group :development, :test do
+  gem "appraisal", github: "thoughtbot/appraisal"
```
Comment (marian13) [inline Gemfile:9]: @nazar-kostyk

Please, leave a comment with an explanation of why this change is needed.

In other words, leave a `NOTE:` comment with links to sources of this solution.

Thanks.

Comment (marian13) [inline Gemfile:9]: @nazar-kostyk 

Please, specify a concrete commit as well.

This way CI will be deterministic even when someone merges new code into `thoughtbot/appraisal`.

Thanks.

Comment (nazar-kostyk) [reply Gemfile]: Thanks! Addressed in [9bbe464](https://github.com/marian13/convenient_service/pull/81/commits/9bbe4646001f16d120bf10659f3168ac5c02ce40)

Comment (nazar-kostyk) [reply Gemfile]: Thanks! Addressed in [9bbe464](https://github.com/marian13/convenient_service/pull/81/commits/9bbe4646001f16d120bf10659f3168ac5c02ce40)

**Review by marian13 — APPROVED** [review]
